packages/compile-common/src/sources.ts

```diff
@@ -1,6 +1,21 @@
-import type { Sources, CollectedSources } from "./types";
 import * as path from "path";
 
+//source content by path
+export interface Sources {
+  [sourcePath: string]: string;
+}
+
+//original paths by transformed path
+export interface PathMapping {
+  [sourcePath: string]: string;
+}
+
+export interface CollectedSources {
+  sources: Sources;
+  targets: string[];
+  originalSourcePaths: PathMapping;
+}
```

Comment:
Why remove these from the `types` file?

Reply:
`Sources` was being clobbered in that other location due to this namespace being named `Sources`. So in other words, these are the exports for everything in compile-common/src...
```
export * as Shims from "./shims";
export * as Sources from "./sources";
export * as Errors from "./errors";
export * as Compilations from "./compilations";
export * from "./types";
```

The `Sources` type was being clobbered by the `Sources` namespace. @gnidan advised me that this is a perfectly good new location.

Reply:
Makes sense. Might be worth adding a comment, but I don't think it's strictly necessary.

Reply:
Probably the only thing useful in the future will be the conversation on this PR. Nobody is going to know that these types used to be in `./types.ts`, right?

Reply:
Yeah fair enough.